app/controllers/courses_controller.rb

```diff
@@ -110,7 +110,7 @@ def requirements
         ["Qualifications needed", "required_qualifications"],
         ["Personal qualities", "personal_qualities"],
         ["Other requirements", "other_requirements"],
-      ].keep_if { |_name, field| copy_field_if_present_in_source_course(field) }
+      ].reject { |_name, field| field == "required_qualifications" && @course.recruitment_cycle_year.to_i > 2021 }.keep_if { |_name, field| copy_field_if_present_in_source_course(field) }
```

Comment:
I would suggust you take this approach 

https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/services/course_creation_step_service.rb#L21-L27

as the cycle is coming to an end imminently and this can be refactor to have 2 items in array